src/wwwroot/styles/views/_settings.scss

```diff
@@ -588,7 +588,10 @@
   .my-plans--container{
     .my-plan--info-container{
       .item {
-        div {margin-top: 1em;}
+        div {          
+          margin-top: 1em;
+          .hide{display: none;}
```

Comment:
There is not a general rule for this? I mean `.hide{display: none;}` at top level, because I imagine that we want to hide all elements marked with _hide_ class.